### PR TITLE
Update de_de.json

### PR DESCRIPTION
**Pull request by Krashgamer**
Updated this

### CODE REVIEW

**Comment by xpple** [comment]
Thank you! It seems you changed the indentation a bit, could you revert that?

**Comment by Krashgamer** [comment]
indentation?

**Comment by xpple** [comment]
Yes, how many whitespaces there are before the "code" begins.

**Comment by Krashgamer** [comment]
ahh ok it seems the german one is different from the english one.. give me a sec

**Comment by Krashgamer** [comment]
I hope this is correct now

**Comment by xpple** [comment]
Yep, looks good! I will have to verify the translations first, then I'll merge.

**Comment by Krashgamer** [comment]
sure thing :D

File: src/main/resources/assets/seedmapper/lang/de_de.json [hunk 1]
```diff
@@ -23,12 +28,30 @@
...
+    "command.highlight.block.noneFound": "Kein %s zum hervorheben konnte gefunden werden.",
+    "command.highlight.block.success": "%d %s wurde hervorgehoben.",
+
+    "command.locate.biome.noneFound": "Konnte %s innerhalb 6400 Blöcken nicht finden.",
+    "command.locate.biome.success.0": "%s gefunden bei ",
+    "command.locate.biome.success.1": "Klicke um die Koordinaten von %s zu kopieren.",
+    "command.locate.feature.structure.noneFound": "Konnte %s innerhalb 6400 Blöcken nicht finden.",
+    "command.locate.feature.structure.success.0": "%s gefunden bei ",
+    "command.locate.feature.structure.success.1": "Klicke um die Koordinaten von %s zu kopieren.",
+    "command.locate.feature.slimeChunk.noneFound": "Konnte Slime Chunks innerhalb 6400 Blöcken nicht finden.",
```
Comment (mariusdkm) [inline src/main/resources/assets/seedmapper/lang/de_de.json:40]: I think in the german translations there is always a `von` missing in front of the `6400 Blöcken`.
So it should be `innerhalb von 6400Blöcken nicht finden.`
Not  `innerhalb 6400 Blöcken nicht finden.`

Comment (Krashgamer) [reply src/main/resources/assets/seedmapper/lang/de_de.json:40]: good call...thank you

**Comment by xpple** [comment]
Thanks!

**Pull request merged**